lib/dream_html.ml

```diff
@@ -865,3 +865,146 @@ module Hx = struct
   let ws_connect fmt = string_attr "data-ws-connect" fmt
   let ws_send = attr "data-ws-send"
 end
+
+module MathML = struct
```

Comment:
Can we copy the layout of the `HTML` module and put the attributes first and the tags second inside the module? Also can we add section header comments to make navigation easier? See eg https://yawaramin.github.io/dream-html/dream-html/Dream_html/HTML/index.html#attributes